src/issue.ts

```diff
@@ -82,11 +87,38 @@ async function getRepoIssues(repo: Repo, flags?: Flags): Promise<IssueResult> {
       url = url + `&page=${pageToken}`;
     }
 
-    try {
-      res = await request<IssuesApiResponse>({url});
-    } catch (e) {
-      console.warn(`Error fetching issues for ${repo.repo}.`);
-      // console.warn(e);
+    let tries = 0;
+    while (tries < nBackoff) {
+      try {
+        console.log(tries);
+        res = await request<IssuesApiResponse>({url});
+        break;
+      } catch (e) {
+        // If DRGHS is not actually tracking issues, log a warn on it and skip; do not
+        // reattempt with backoff
+        if (
+          e.response.status === 404 &&
+          e.response.data.message ===
+            `repository ${repo.repo} is not tracking issues`
+        ) {
+          console.warn(
+            `Repository ${repo.repo} is not tracking issues in DRGHS... skipping.`
+          );
+          return result;
```

Comment:
This return doesn't seem right.  Is this supposed to short-circuit exit the function?

Reply:
That was the idea. If drghs claims it is not tracking that repository, then warn and skip processing that repository all-together.